faqs/questions.mdx

```diff
@@ -74,7 +74,7 @@ Yes we do, check it out [here](https://novustatus.com/).
 
 We are now building an automatic tool to help with this migration process, for now, feel free to ask us over Discord, or at engineering@novu.co.
 
-## Pricing
+# Pricing
```

Comment:
In my opinion ## was correct here

because now # --> ###. It is going H1 -> H3 which is not correct

Reply:
Okay.